doc/man1/openssl-fipsinstall.pod.in

```diff
@@ -47,6 +48,12 @@ This indicates if the Known Answer Self Tests (KAT's) have successfully run.
 
 =item - A MAC of the status indicator.
 
+=item - A control to permit PKCS#1 version 1.5 padding.
+
+By default, the FIPS module will accept PKCS#1 version 1.5 padding.
+The default value of '1' disallows such padding.
```

Comment:
Yep this is confusing..

Reply:
I think I got this wrong, the default of 1 allows such.